packages/kbn-es/src/serverless_resources/project_roles/security/roles.yml

```diff
@@ -46,7 +46,7 @@ viewer:
         - feature_siem.read
         - feature_siem.read_alerts
         - feature_siem.endpoint_list_read
-        - feature_securitySolutionCases.read
+        - feature_securitySolutionCasesV2.read
```

Comment:
Are these new privileges for all these roles also updated in the [elasticsearch-controller](https://github.com/elastic/elasticsearch-controller/blob/main/helm/values.yaml) repo? The privileges in this file here should be a copy of those. If not, I'd suggest you create a PR on the other repo first, to update the privileges before merging this PR.

Reply:
The privileges in [`roles.yml`](https://github.com/elastic/kibana/blob/631ccb031ca59d51b5db0939cf6327e36e5a34b3/packages/kbn-es/src/serverless_resources/project_roles/security/roles.yml#L2), [`security_roles.json`](https://github.com/elastic/kibana/blob/631ccb031ca59d51b5db0939cf6327e36e5a34b3/packages/kbn-es/src/serverless_resources/security_roles.json#L1), [`roles.yml`](https://github.com/elastic/kibana/blob/631ccb031ca59d51b5db0939cf6327e36e5a34b3/x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/serverless/es_serverless_resources/roles.yml#L17) and [`project_controller_security_roles.yml` ](https://github.com/elastic/kibana/blob/631ccb031ca59d51b5db0939cf6327e36e5a34b3/x-pack/test_serverless/shared/lib/security/kibana_roles/project_controller_security_roles.yml#L2)need to be in sync with the ones in `elasticsearch-controller` for our testing needs.